examples/transactions/transaction_basic_example.py

```diff
@@ -2,17 +2,11 @@
 import time
 
 client = hazelcast.HazelcastClient()
-transaction = client.new_transaction(timeout=10)
-try:
-    transaction.begin()
-    transactional_map = transaction.get_map("transaction-map")
-    print("Map: {}".format(transactional_map))
+
+with client.new_transaction(timeout=10) as transaction:
+    transactional_map = transaction.get_map("transactional-map")
+    print("Created map:", transactional_map)
 
     transactional_map.put("1", "1")
     time.sleep(0.1)
     transactional_map.put("2", "2")
-
-    transaction.commit()
-except Exception as ex:
-    transaction.rollback()
-    print("Transaction failed! {}".format(ex.args))
```

Comment:
Just curious, can users add exception handler like this with context manager approach?

Reply:
they can do something like 
```py
with ... as smth:
    try:
        ...
    except ..:
       ...
```